tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -130,7 +130,7 @@ class CompletionArgSuite extends BaseCompletionSuite {
     s"""|
         |$user
         |object Main {
-        |  User("", @@ address = "")
+        |  User("", @@, address = "")
```

Comment:
This will not work currently without the comma, but I'm not sure if that's an important case cover.
It didn't work before those changes as soon as the user started to type smth.

Reply:
We would have to add `,` manually to make it work then. I think it's safe to change this